src/Steppenwolf/Steppenwolf.csproj

```diff
@@ -3,13 +3,15 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <CodeAnalysisRuleSet>../../stylecop.ruleset</CodeAnalysisRuleSet>
+    <DisableImplicitComponentsAnalyzers>true</DisableImplicitComponentsAnalyzers>
```

Comment:
DisableImplicitComponentsAnalyzers ?

Reply:
Needed due to an issue with `Microsoft.AspNetCore.Components.Analyzers.dll : Assembly with same name is already loaded`